.rubocop.yml

```diff
@@ -163,7 +163,7 @@ InternalAffairs/StyleDetectedApiUse:
   Exclude:
     - 'lib/rubocop/cop/mixin/percent_array.rb'
 
-InternalAffairs/NumblockComplement:
+InternalAffairs/NumblockHandler:
```

Comment:
Might be good to reorder the commits to avoid polluting the history with this rename. I also not super fond of the name, but as an internal cop it's not that big of a deal.

Reply:
Agreed about refactoring away the rename.